src/vcpkg/visualstudio.cpp

```diff
@@ -169,6 +169,7 @@ namespace vcpkg::VisualStudio
         // VS 2017 changed the installer such that cl.exe cannot be found by path navigation and
         // the env variable is only set when vcvars has been run. Therefore we close the safety valves.
         maybe_append_comntools("vs160comntools", "16.0", false);
+        maybe_append_comntools("VS170COMNTOOLS", "17.0", false);
```

Comment:
```suggestion
        maybe_append_comntools("vs170comntools", "17.0", false);
        maybe_append_comntools("vs160comntools", "16.0", false);
```

1. Keep the existing 'reversed' order?
2. The other ones are lowercase...

If there's some reason this one has to be different than the others it probably deserves a comment?